### PR TITLE
Add documentation for Cypress Test Data Generator plugin

### PR DESCRIPTION
**Pull request by khawjaahmad**

### CODE REVIEW

File: src/data/plugins.json
Comment (jennifer-shehane) [inline src/data/plugins.json:435]: @khawjaahmad This link does not work. Can you link to the plugin?

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]

**Comment by MikeMcC399** [comment]
@khawjaahmad 

I noticed some issues with your repository 
- https://github.com/khawjaahmad/cypress-test-data-generator

and your published package
- https://www.npmjs.com/package/cypress-test-data-generator

so I will open some issues in your repo to help you fix the issues.

As far as your entry in the [Plugins > Development Tools](https://docs.cypress.io/plugins#development-tools) section of the Cypress Documentation is concerned, you should move it to the end of the Community list and before the Deprecated list, as mentioned in the guide [CONTRIBUTING](https://github.com/cypress-io/cypress-documentation/blob/main/CONTRIBUTING.md#adding-plugins)

> Plugins are listed in the following order:
>
> - official (Cypress owned)
> - verified (community owned and verified by Cypress)
> - community (community owned and unverified)
> - deprecated (npm registry missing, source repo archived or incompatible with v10+)

**Comment by MikeMcC399** [comment]
The issues which need addressing are listed above.

**Comment by khawjaahmad** [comment]
@MikeMcC399 

I have done the changes as requested and marked the issues closed. Also updated the plugin entry in the correct order.
Thanks

**Review by khawjaahmad — COMMENTED** [review]
changes addressed.

**Comment by MikeMcC399** [comment]
@khawjaahmad 

> I have done the changes as requested and marked the issues closed. Also updated the plugin entry in the correct order. 

Thank you for the confirmation. Your npm module now looks in good shape.

You will need to wait for re-review by the Cypress.io team now. (As an external contributor I don't have privileges to merge your content.)

**Review by jennifer-shehane — APPROVED** [review]

**Pull request merged**
